tests/fixtures/tables/autodetect.sql

```diff
@@ -0,0 +1,53 @@
+DROP SCHEMA IF EXISTS autodetect CASCADE;
+CREATE SCHEMA autodetect;
+
+CREATE TABLE autodetect.auto_table
+(
+    gid     SERIAL PRIMARY KEY,
+    feat_id INTEGER,
+    geom    GEOMETRY(POINT, 4326)
+);
+
+-- INSERT INTO autodetect.auto_table
+-- SELECT generate_series(1, 3) as id,
+--        (random() * 100000)::int as feat_id,
+--        (ST_DUMP(ST_GENERATEPOINTS(st_transform(st_tileenvelope(18, 235085, 122323), 4326), 3))).geom;
+-- INSERT INTO autodetect.auto_table
+-- SELECT generate_series(4, 30) as id,
+--        (random() * 100000)::int as feat_id,
+--        (ST_DUMP(ST_GENERATEPOINTS(st_transform(st_tileenvelope(0, 0, 0), 4326), 27))).geom;
+
```

Comment:
Nit: Is this comment still needed?

Reply:
Yes, all other tests include the code how they were initially generated. Makes it easier to regen if needed, plus it documents ranges